lib/exabgp/configuration/setup.py

```diff
@@ -230,6 +230,12 @@
 		},
 	},
 	'api':  {
+		'lines': {
```

Comment:
Ack, So the name I meant to give it was chunk, as in this controls how the size of the chunk of data being sent back before yield is. Apparently instead I ended up calling the 'read' property 'chunk' instead.  This is what happens when i write code while tired :-)

So do you prefer, 'chunk', 'lines' or something else for the environment variable name?